Library/OcAppleImg4Lib/libDERImg4/libDERImg4_config.h

```diff
@@ -18,6 +18,23 @@
 extern "C" {
 #endif
 
+#define DER_IMG4_MAX_DIGEST_SIZE  64
+
+  extern const uint8_t *DERImg4RootCertificate;
```

Comment:
Why such strange indentation out of a sudden? Move it left in my opinion… I do not see anywhere else extern causing moved indentations.

Also, how is the root certificate going to be installed? Different models have different certificates (at least I thought so), and we want to be able to switch between models via configuration.

Reply:
I thought there was only one Root Certificate. Can we verify this?

Reply:
Probably cannot, but even if we can I can imagine them inventing another for some newer models in the future. What could we do with that? Maybe make this a function? I believe we can use only one certificate at once, but I am not positive only one certificate exists.

Reply:
The library does not mandate the form of passing, so, maybe you do not like that style, but it could be defined as a macro resolving to a function call. Would we need to pass a context?

Reply:
To be honest, maybe a global state is ok, we can just make some function to update the certificate if and when it is changed in the future. Mark as resolved if you are fine with it.

Reply:
This is a very small detail, we can adapt it as needed when we need more flexibility.